homeassistant/components/unifi/sensor.py

```diff
@@ -140,6 +140,12 @@ def async_device_outlet_supported_fn(hub: UnifiHub, obj_id: str) -> bool:
     return hub.api.devices[obj_id].outlet_ac_power_budget is not None
 
 
+@callback
+def async_get_password_supported_fn(hub: UnifiHub, obj_id: str) -> bool:
+    """Determine if a WLAN supports reading the password."""
+    return hasattr(hub.api.wlans[obj_id], "x_passphrase")
```

Comment:
This will always return true as it's a property method https://github.com/Kane610/aiounifi/blob/95761ee496ddfb1c8e05695e50fb9061a1552956/aiounifi/models/wlan.py#L288

You can probably make this a lambda in the entity description if it fits on one line

Reply:
Done!